.github/workflows/sonar-cloud.yml

```diff
@@ -0,0 +1,27 @@
+name: SonarCloud
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    tags: [v*]
```

Comment:
I can't find anything about support for tags in the sonarcloud documenation. It does support branch analysis (https://docs.sonarsource.com/sonarcloud/enriching/branch-analysis/) so I think it makes more sense to run this on the `release-*` branches.